protocol/messenger_profile_showcase.go

```diff
@@ -206,6 +207,10 @@ func (m *Messenger) SetProfileShowcasePreferences(preferences *ProfileShowcasePr
 		return err
 	}
 
+	return m.DispatchProfileShowcase()
+}
+
+func (m *Messenger) DispatchProfileShowcase() error {
```

Comment:
Hey, just curious, was it really necessary to replace the `m.publishContactCode()` with a function containing a single function call to `m.publishContactCode()` ?

Reply:
In the next PR i'm going to add more `DispatchProfileShowcase` calls in communities and contacts sections. And it's not obvious that `m.publishContactCode` should be invoked to dispatch profile showcase message. And in future, if we decide to separate contact code and profile showcase message it's better to have just one place to modify